yamllint/config.py

```diff
@@ -20,6 +20,13 @@
 import yaml
 
 import yamllint.rules
+from yamllint import conf
+
+try:
+    import importlib.resources as pkg_resources
+except ImportError:
+    # Try backported to PY<37 `importlib_resources`.
+    import importlib_resources as pkg_resources
```

Comment:
New dependencies cause other problems. Even if this one is part of standard lib after Python 3.9, I would prefer not adding a new one as long as we support Python ⩽ 3.8.

Is it considerable to do something like this?
```python
try:
    import importlib.resources
    _PATH = importlib.resources.resource_filename(...)
except ImportError:
    _PATH = os.path.dirname(os.path.realpath(__file__)```

Reply:
It would be a bit ugly, but something like the following would work:

```python
try:
    import importlib.resources
    _PATH = importlib.resources.resource_filename(...)
except ImportError:
    try:
        import importlib_resources
        _PATH = importlib_resources.resource_filename(...)
    except ImportError:
        _PATH = os.path.dirname(os.path.realpath(__file__)
```

This way importlib_resources could be manually installed in order to make packaging successful for Python <3.9.  Or importlib_resources could be added as a "extra" dependency.

Reply:
I really really prefer to avoid new dependencies. Good news is that `importlib.resources` is available for recent Python. For old versions, well, too bad...

Reply:
Ok, dropped the new dep entirely